### PR TITLE
Feat:  finalize message component

### PR DESCRIPTION
**Pull request by alamorre**
This is not ready yet, I'll ping ya when it is.

### CODE REVIEW

**Comment by alamorre** [comment]
Kk Lets walk through this tomorrow!

This is gonna make you 🤤🤤🤤

File: src/ChatFeed/MessageList/Message/index.tsx [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+            attachment={attachment}
+            isLoading={isSending || attachment.file === null}
+            style={{ ...styles.file, ...style.file }}
+          />
+        );
+      }
+
+      return <div key={`attachment${index}`} />;
+    });
+  };
```
Comment (dhavelock) [inline src/ChatFeed/MessageList/Message/index.tsx:99]: maybe combine these methods and pass an boolean to pick if it should render images and files- other than the if clause and style they are identical

**Review by dhavelock — APPROVED** [review]

File: src/ChatFeed/MessageList/Message/index.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+  const paddingBottom =
+    !nextMessage || nextMessage.sender_username !== message.sender_username
+      ? '12px'
+      : '2px';
+  const styles = isMyMessage ? myStyles : theirStyles;
+  const sendingStyle = isSending
+    ? {
+        backgroundColor: isSending ? '#40a9ff' : 'rgb(24, 144, 255)',
+      }
+    : {};
```
[the remark this answers is not in the document]
Comment (alamorre) [reply src/ChatFeed/MessageList/Message/index.tsx:52]: Ah yes, totally redundant, fixing...

File: src/ChatFeed/MessageList/Message/index.tsx (hunk 1 above)
Comment (alamorre) [reply src/ChatFeed/MessageList/Message/index.tsx:99]: Happy to do that - love chopping line count down 👍

**Pull request merged**